src/client/components/modals/ConfirmDeleteModal.tsx

```diff
@@ -28,10 +28,24 @@ const ConfirmDeleteModal: React.FC<ConfirmDeleteModalProps> = ({ isOpen, setOpen
       </ModalBody>
       <ModalFooter>
         <Flexbox direction="row" className="w-full justify-end" gap="2">
-          <LoadingButton block color="danger" onClick={submitDelete}>
+          <LoadingButton
```

Comment:
The only other component using ConfirmDeleteModal is the Blog Delete action which I tested and works the same. The delete button on a blog doesn't live in any other clickable area, so nothing else was listening for the click action.